config/default/views.view.foia_requests.yml

```diff
@@ -892,7 +892,7 @@ display:
                 title: 'Queued for submission'
                 operator: '='
                 value:
-                  value: '0'
+                  value: '3'
```

Comment:
Do you know why this got changed to 3?

Reply:
After updating the email transit to delete after 7 days when I save the node it doesn't save it's asks for option value. In group 1 option value was empty I couldn't able to save it. There were already option value was 1,2,4 so I updated 3 and saved it.

Reply:
@sundarcreosen I have tried it locally and I see what you're saying. However the correct value is 0. Please change it to 0 instead of 3.